pinot-core/src/main/java/com/linkedin/pinot/core/operator/aggregation/function/AggregationFunctionFactory.java

```diff
@@ -69,6 +73,9 @@ public static AggregationFunction getAggregationFunction(String functionName) {
       case DISTINCTCOUNTHLL_AGGREGATION_FUNCTION:
         return new DistinctCountHLLAggregationFunction();
 
+      case FASTHLL_AGGREGATION_FUNCTION:
+        return new FastHllAggregationFunction(segmentMetadata.getStarTreeMetadata().getHllLog2m());
```

Comment:
I will pull hll config in metadata out of StarTreeMetadata into segment metadata in next PR, so it will resolve this issue.